applications/serial_lte_modem/src/slm_at_socket.c

```diff
@@ -257,15 +253,6 @@ static int do_socket_close(void)
 		return 0;
 	}
 
-#if defined(CONFIG_SLM_NATIVE_TLS)
-	if (sock.type == SOCK_STREAM && sock.sec_tag != INVALID_SEC_TAG) {
-		ret = slm_tls_unloadcrdl(sock.sec_tag);
-		if (ret < 0) {
-			LOG_WRN("Fail to unload credential: %d", ret);
-		}
-		sock.sec_tag = INVALID_SEC_TAG;
-	}
-#endif
```

Comment:
Same remark when closing the socket, the current behavior will be to overwrite the other loaded credentials if 2 are in use instead of reusing this one.

Reply:
I'd rather not complicate this. Most use cases only need 1 set of loaded credentials. If there is a more exotic use case, the amount of buffers can be raised as a preliminary cure and this can then be implemented.